api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -235,6 +260,16 @@ message TlsCertificate {
   // error.
   PrivateKeyProvider private_key_provider = 6;
 
+  // [#not-implemented-hide:]
```

Comment:
Should this message itself contain the private key shared among the providers?

Reply:
I want the provider to use the private key here https://github.com/envoyproxy/envoy/blob/c49f392561ba8c246f845d61e555c9536d5edbea/api/envoy/extensions/transport_sockets/tls/v3/common.proto#L200